core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/Parameter.java

```diff
@@ -86,7 +86,7 @@ public String getGroup() {
 	/**
 	 * @return the choice
 	 */
-	public String getChoice() {
+	public String[] getChoice() {
```

Comment:
Will check for it but I supposed that previously we were transforming the attribute in the tooling.